minard/standard_runs.py

```diff
@@ -0,0 +1,50 @@
+import couchdb
+from itertools import groupby
+from .views import app
+from time import time
+from uuid import uuid4
+
+def get_standard_runs():
+    couch = couchdb.Server("https://snoplus:"+app.config["COUCHDB_PASSWORD"]+"@"+app.config["COUCHDB_HOSTNAME"])
+    orca_db = couch['orca']
+    sr_view = orca_db.view("standardRuns/getStandardRunsWithVersion")
+    # standard run keys are [doc-version, run name, run version, timestamp]
+    # I want to groupby by run name then within that group all run versions sorted
+    # by timestamp
+    rows = sorted(sr_view.rows, key=lambda x: x.key[1] + x.key[2])
```

Comment:
I think here it would probably be best *only* to show the latest version (or even better have a drop down menu on the standard runs page which allows you to select which standard run version). The reason is that there are a lot of older standard runs from previous versions that don't show up in Orca but do show up on the page and I think that might be confusing.

Reply:
Ah yeah, I had that at first but  I removed it so I could test on old unused runs. I'll add it back in.